src/styles/PPLoadingSpinner.css

```diff
@@ -1,6 +1,10 @@
 .pp-loading-spinner {
-  width: 40px;
-  height: 40px;
+  width: 80px;
+  height: 80px;
```

Comment:
Spinner is 80px horizontally, not 40px. The .pp-loading-spinner class was horizontally centered but only accounted for 40px.